src/components/Card/index.jsx

```diff
@@ -27,8 +25,18 @@ function Card({ movie, isPoster }) {
     clearTimeout(timeOutId);
   }
 
+  async function selectMovie() {
+    const selectedMovie = await fetchFrom('/movie/', { movieId: movie.id });
+    setInStorage('selectedMovie', selectedMovie);
+    setMovieInfo(selectedMovie);
+  }
+
   return (
-    <div className="w-48 flex-none origin-center ease-out duration-100 hover:duration-300 hover:scale-125">
+    <div
+      className="card w-48 flex-none origin-center ease-out duration-100 hover:duration-300 hover:scale-125"
+      onClick={selectMovie}
+      aria-hidden="true"
+    >
       <img
```

Comment:
instead of rendering an element on mouse hover, it will be more performant to how the overlay on hover with css